### PR TITLE
bump version to v1.0.0rc0

### PR DESCRIPTION
**Pull request by RunningLeon**
## Motivation

Bump version 1.0.0rc0

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/01-how-to-build/macos-arm64.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 ```bash
 # install pytoch & mmcv
-conda install pytorch==1.9.0 torchvision==0.10.0 -c pytorch
+conda install pytorch==1.9.0 torchvision==1.0.0rc0 -c pytorch
```
Comment (lvhan028) [inline docs/zh_cn/01-how-to-build/macos-arm64.md:41]: torchvision 1.0.0rc0?

**Comment by lvhan028** [comment]
Currently, we haven't cherry-picked several PRs related to SDK from master to dev-1.x yet.
I think we can keep it as `0.10.0` and upgrade it to the master version in its next release.
What's your opinion? @lzhangzz

**Comment by irexyc** [comment]
Could you please fix the wrong config path used in [link1](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/docs/zh_cn/04-supported-codebases/mmaction2.md) and [link2](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/docs/en/04-supported-codebases/mmaction2.md)?

`video-recognition_2d_onnxruntime_static.py` should be `video-recognition_onnxruntime_static.py`

**Review by grimoire — DISMISSED** [review]

File: docs/zh_cn/01-how-to-build/macos-arm64.md [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
 ```bash
 # install pytoch & mmcv
-conda install pytorch==1.9.0 torchvision==0.10.0 -c pytorch
+conda install pytorch==1.9.0 0.10.0 -c pytorch
```
Comment (lvhan028) [inline docs/zh_cn/01-how-to-build/macos-arm64.md:41]: `torchvision` is missing

**Review by grimoire — APPROVED** [review]

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**